firebase-functions/src/main/java/com/google/firebase/functions/FirebaseFunctions.java

```diff
@@ -191,6 +195,18 @@ public void useFunctionsEmulator(String origin) {
     urlFormat = origin + "/%2$s/%1$s/%3$s";
   }
 
+  @RestrictTo(RestrictTo.Scope.TESTS)
+  @VisibleForTesting
+  void replaceAuthorizationHeaderForTest(String headerName) {
+    authorizationHeader = checkNotNull(headerName, "Null header name");
+  }
+
+  @RestrictTo(RestrictTo.Scope.TESTS)
+  @VisibleForTesting
+  void replaceIidHeaderForTest(String headerName) {
```

Comment:
Surely this isn't necessary? The IID token isn't used for anything automatically.